arduino-ide-extension/src/electron-main/theia/electron-main-application.ts

```diff
@@ -23,8 +23,10 @@ import * as os from '@theia/core/lib/common/os';
 import { Restart } from '@theia/core/lib/electron-common/messaging/electron-messages';
 import { TheiaBrowserWindowOptions } from '@theia/core/lib/electron-main/theia-electron-window';
 import { IsTempSketch } from '../../node/is-temp-sketch';
+import { CLOSE_PLOTTER_WINDOW, SHOW_PLOTTER_WINDOW } from '../../common/ipc-communication';
 
 app.commandLine.appendSwitch('disable-http-cache');
+app.commandLine.appendSwitch('disable-site-isolation-trials');
```

Comment:
Why is it needed?

Reply:
My bad, this was a leftover from a previous solution 🙏